app/controllers/CyaMaintainProfileController.scala

```diff
@@ -104,6 +103,38 @@ class CyaMaintainProfileController @Inject() (
       case Left(errors) =>
         Future.successful(logErrorsAndContinue(errorMessage, routes.ProfileController.onPageLoad(), errors))
     }
+  }
+
+  def onPageLoadNirmsNumber(): Action[AnyContent] = (identify andThen getData andThen requireData) { implicit request =>
```

Comment:
on page load method for nirms number cya - includes nirms number in summary list